### DISCUSSION TITLE
How are recognition model performance evaluation metrics achieved in recognition?

### DISCUSSION THREAD

**Discussion by yerin723** [mindee/doctr#1325] [General]
We are comparing the performance of the existing pretrained weight and the trained model.
How are recognition performance evaluation indicators achieved? Which one do you use,
CER (Character Error Rate) or WER (Word Error Rate)? What is the reason?

**Comment by felixT2K** [comment]
Hi @yerin723 👋🏼,

You can use https://github.com/mindee/doctr/blob/main/references/recognition/evaluate_pytorch.py to benchmark your model (currently we use FUNSD and CORD for benchmarking)

**Comment by yerin723** [comment]
How are exact_match, partial_match result values ​​inferred?

**Reply by felixT2K** [reply]
implementation: https://github.com/mindee/doctr/blob/28e5375c399ae5dd146a1b7e13bd5831acb6dc2d/doctr/utils/metrics.py#L47

exact =  word1 == word2
partial = unidecode(word1).lower() == unidecode(word2).lower()

sum up and divide by total :)
